src/Keycloak.Dotnet.Sdk/Dtos/Realms/RealmDto.cs

```diff
@@ -0,0 +1,137 @@
+
+    [JsonPropertyName("supportedLocales")]
+    public string[]? SupportedLocales { get; set; }
+
+    [JsonPropertyName("defaultLocale")]
+    public string? DefaultLocale { get; set; }
+
+    [JsonPropertyName("identityProviders")]
+    public IdentityProviders[]? IdentityProviders { get; set; }
+
+    [JsonPropertyName("identityProviderMappers")]
+    public IdentityProviderMappers[]? IdentityProviderMappers { get; set; }
+
+    [JsonPropertyName("attributes")]
+    public Dictionary<string, string>? Attributes { get; set; }
+}
+
+public record IdentityProviders
+{
+    [JsonPropertyName("alias")]
+    public string? alias { get; set; }
+
+    [JsonPropertyName("internalId")]
+    public string? internalId { get; set; }
+
+    [JsonPropertyName("providerId")]
+    public string? providerId { get; set; }
+
+    [JsonPropertyName("enabled")]
+    public bool? enabled { get; set; }
+
+    [JsonPropertyName("updateProfileFirstLoginMode")]
+    public string? updateProfileFirstLoginMode { get; set; }
+
+    [JsonPropertyName("trustEmail")]
+    public bool? trustEmail { get; set; }
+
+    [JsonPropertyName("storeToken")]
+    public bool? storeToken { get; set; }
+
+    [JsonPropertyName("addReadTokenRoleOnCreate")]
+    public bool? addReadTokenRoleOnCreate { get; set; }
+
+    [JsonPropertyName("authenticateByDefault")]
+    public bool? authenticateByDefault { get; set; }
+
+    [JsonPropertyName("linkOnly")]
+    public bool? linkOnly { get; set; }
+
+    [JsonPropertyName("firstBrokerLoginFlowAlias")]
+    public string? firstBrokerLoginFlowAlias { get; set; }
+
+    [JsonPropertyName("config")]
+    public IdentityProvidersConfig? config { get; set; }
+}
+
+public record IdentityProvidersConfig
+{
+    [JsonPropertyName("clientId")]
+    public string? ClientId { get; set; }
+
+    [JsonPropertyName("tokenUrl")]
+    public string? TokenUrl { get; set; }
+
+    [JsonPropertyName("authorizationUrl")]
+    public string? AuthorizationUrl { get; set; }
+
+    [JsonPropertyName("clientAuthMethod")]
+    public string? ClientAuthMethod { get; set; }
+
+    [JsonPropertyName("syncMode")]
+    public string? SyncMode { get; set; }
+
+    [JsonPropertyName("clientSecret")]
+    public string? ClientSecret { get; set; }
+
+    [JsonPropertyName("useJwksUrl")]
+    public string? UseJwksUrl { get; set; }
+}
+
+public record IdentityProviderMappers
+{
+    [JsonPropertyName("id")]
+    public string? Id { get; set; }
+
+    [JsonPropertyName("name")]
+    public string? Name { get; set; }
+
+    [JsonPropertyName("identityProviderAlias")]
+    public string? IdentityProviderAlias { get; set; }
+
+    [JsonPropertyName("identityProviderMapper")]
+    public string? IdentityProviderMapper { get; set; }
+
+    [JsonPropertyName("config")]
+    public IdentityProviderMappersConfig? Config { get; set; }
+}
+
+public record IdentityProviderMappersConfig
+{
+    [JsonPropertyName("syncMode")]
+    public string? SyncMode { get; set; }
+}
```

Comment:
Same pattern as in other records. Or subclasses with Composition suffix, or refact other places to use like this.

Reply:
Done